runelite-client/src/main/java/net/runelite/client/plugins/camera/CameraPlugin.java

```diff
@@ -66,6 +71,11 @@
 	private static final String LOOK_SOUTH = "Look South";
 	private static final String LOOK_EAST = "Look East";
 	private static final String LOOK_WEST = "Look West";
+	private static final int RESET_DIRECTION_TIMEOUT = 5;
+	private int lastUpdatedTick = 0;
+	private static final String defaultLeftClickOption = LOOK_NORTH;
+	private String currentLeftClickOption;
+	private String[] options = new String[]{LOOK_NORTH, LOOK_SOUTH, LOOK_EAST, LOOK_WEST};
```

Comment:
This can be made `static final` and moved with the other constants.
```suggestion
	private static final String[] OPTIONS = new String[]{LOOK_NORTH, LOOK_SOUTH, LOOK_EAST, LOOK_WEST};
```

Reply:
Also NSEW is a strange order, I would expect to see this in NESW order.

Reply:
That's the order that the current Camera Options plugin shows options, so I didn't want to do different than that. I can change this to NESW though.

Reply:
Hm, that's fine then. Let's leave it be.